pages/linux/fatrace.md

```diff
@@ -0,0 +1,12 @@
+# fatrace
+
+> Report file access events.
+> More information: <https://manned.org/fatrace>.
+
+- Print file access events in all mounted filesystems to `stdout`:
+
+`sudo fatrace`
+
+- Print file access events in the [c]urrent directory, with [t]imestamps, to `stdout`:
+
+`sudo fatrace -c -t`
```

Comment:
There are also long forms of these options, so we may want to include them in the example. Let me know what you think.
```suggestion
- Print file access events in the [c]urrent directory, with [t]imestamps, to `stdout`:

`sudo fatrace --current-mount --timestamp`
```
We can use the placeholder showing both short and long form, or the short option mnemonics, but we rather not use both. 
```suggestion
- Print file access events in the current directory, with timestamps, to `stdout`:

`sudo fatrace {{-c|--current-mount}} {{-t|--timestamp}}`
```

Reply:
Seeing it like this, showing both short and long form definitely seems more instructive. Now reading back on the [option syntax style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax) it seems more appropiate.

I personally like seeing the short form, so if it's good with you I'm committing the latter version.

I also just realised that my message was not actually accurate: `--current-mount` really is the mount of the current directory and does not filter it down to just the current directory